### DISCUSSION TITLE
Docker development on Vs-code-server

### DISCUSSION THREAD

**Discussion by Freddy-0** [coder/code-server#3649] [Q&A] [answered]
I currently have code-server set up and working fine, but when tryint to install the docker extension it wont install docker. Is this due to it already running on docker or is docker generally not supported on remote sessions?
I need docker for development, but i cant find any documentation about this here. Help would be much appriciated

**Comment by jsjoeio** [comment]
Hmm...someone else reported [an issue](https://github.com/cdr/code-server/issues/2756) back in March. Feel free to post reproduction steps there and we can try and take a look!

**Comment by ghost** [comment]
What about [docker in code-server container?](https://github.com/cdr/code-server/blob/main/docs/FAQ.md#docker-in-code-server-container)

**Comment by aptalca** [comment, marked as answer]
Just fyi, the [linuxserver code-server image](https://github.com/linuxserver/docker-code-server) has [a mod for accessing docker on host](https://github.com/linuxserver/docker-mods/tree/universal-docker), and [another mod for running docker-in-docker](https://github.com/linuxserver/docker-mods/tree/universal-docker-in-docker) inside code server (among a bunch of other mods: https://mods.linuxserver.io/?mod=code-server)
